addons/l10n_cl/views/report_invoice.xml

```diff
@@ -268,4 +318,14 @@
         </tr>
     </template>
 
+    <template id="yielding_invoice">
+        <t t-call="web.html_container">
+            <t t-foreach="docs" t-as="o">
+                <t t-call="l10n_cl.report_invoice_document">
+                    <t t-set="show_info_box" t-value="True"/>
+                </t>
+            </t>
+        </t>
```

Comment:
Normally you shouldn't need to use the redefinition of the o variable if you target the right template :thinking: check l10n_cl.report_invoice instead of report_invoice_document.
```suggestion
        <t t-call="l10n_cl.report_invoice">
            <t t-set="show_info_box" t-value="True"/>
        </t>
```